core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -154,7 +161,8 @@ private[spark] class CoarseMesosSchedulerBackend(
     startScheduler(driver)
   }
 
-  def createCommand(offer: Offer, numCores: Int, taskId: String): CommandInfo = {
+  def createCommand(offer: Offer, numCores: Int, taskId: String, randPorts: List[Long] = List())
```

Comment:
I think according to spark style we should move the method parameters into each line in this case.